metadata.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "puppetlabs-reboot",
-  "version": "0.1.10-SNAPSHOT",
+  "version": "0.2.0-SNAPSHOT",
```

Comment:
This is a breaking change so I figured we should bump the version number.  Honestly, I think once we merge this and #59, we should bump the version to 1.0.0.